docs/Catalog/Preferences.md

```diff
@@ -48,16 +48,21 @@ drag-and-drop or from folders in S3
 * `ui.blocks.browser: False` - hide files browser on both Bucket and Packages tab
 * `ui.blocks.code: False` - hide Code block with quilt3 code boilerplate
 * `ui.blocks.meta: False` - hide Metadata block on Package page
-* `ui.sourceBuckets` - a dictionary of S3 bucket names that map to an empty object reserved for future enhancements;
+* `ui.sourceBuckets` - a dictionary of S3 bucket names
+that map to an empty object reserved for future enhancements;
 buckets in this dictionary are the ones offered when the user clicks
-Revise Package > Add files from Bucket; if the dictionary is not set or is empty the feature "Add files from Bucket" is disabled
-* `ui.defaultSourceBucket` - source bucket from `ui.sourceBuckets` that is selected by default; if it doesn't match any bucket then it's ignored
-* `ui.package_description` - a dictionary that maps package handle regular expressions or literals to JSONPath expressions of fields to show from package metadata in the package list view.
+Revise Package > Add files from Bucket; if the dictionary is not set
+or is empty the feature "Add files from Bucket" is disabled
+* `ui.defaultSourceBucket` - source bucket from `ui.sourceBuckets`
+that is selected by default; if it doesn't match any bucket then it's ignored
+* `ui.package_description` - a dictionary
```

Comment:
I think GitBook doesn't work well with multi-line list items: see 16aad503cffcc57d86b1145fb8df2ce897415773 and f152a237f85988713cfe6ade77c2ae93dd6a8783.
I'm not sure what useful markdownlint does besides that.

Reply:
Already existed multiline list handled https://docs.quiltdata.com/catalog/preferences#properties
Maybe it was an old issue, that was fixed in GitBook?

Reply:
linter is useful to not think about formatting anymore, and to not have unnecessary changes introduced by formatting